Content.Client/Power/APC/ApcVisualizerComponent.cs

```diff
@@ -0,0 +1,107 @@
+using Content.Shared.APC;
+
+namespace Content.Client.Power.APC;
+
+[RegisterComponent]
+[Access(typeof(ApcVisualizerSystem))]
+public sealed class ApcVisualsComponent : Component
```

Comment:
Shouldn't this use the component?

Reply:
The component is in the server and moving it to shared would require moving BaseNetConnectorComponent to shared as well as other power stuff which i think its out of scope for this pr